cookbook/frontend/assetic_php.rst

```diff
@@ -0,0 +1,152 @@
+.. index::
+    single: Front-end; Assetic, Bootstrap
+
+Combining, Compiling and Minimizing Web Assets with PHP Libraries
+=================================================================
+
+The official Symfony Best Practices recommend to use Assetic to
+:doc:`manage web assets </best_practices/web-assets>`, unless you are
+comfortable with JavaScript-based frontend tools.
+
+Even if those JavaScript-based solutions are the most suitable ones from a
+technical point of view, using pure PHP alternative libraries can be useful in
+some scenarios:
+
+* If you can't install or use ``npm`` and the other JavaScript solutions;
+* If you prefer to limit the amount of different technologies used in your
+  applications;
+* If you want to simplify application deployment.
+
+In this article you'll learn how to combine and minimize CSS and JavaScript files
```

Comment:
well you agreed which is good. So it is not "it must" but it is optional. If we make this into a rule just to be always right is ok. I personally think this interlude is so short that flows immediately to the point, no need for refrain or pause. I also doubt this introduces any confusion. But :green_heart: :traffic_light:  light if you always want to be right. It is just i am concern on the damage to the language over time with enforcing something that is optional indeed. But yeah let's go in agreement to put the comma :older_man: